conans/client/conan_command_output.py

```diff
@@ -82,13 +86,120 @@ def _read_dates(self, deps_graph):
     def nodes_to_build(self, nodes_to_build):
         self.user_io.out.info(", ".join(str(n) for n in nodes_to_build))
 
+    def _handle_json_output(self, data, json_output, cwd):
+        json_str = json.dumps(data)
+
+        if json_output is True:
+            self.user_io.out.write(json_str)
+        else:
+            if not os.path.isabs(json_output):
+                json_output = os.path.join(cwd, json_output)
+            save(json_output, json.dumps(data))
+            self.user_io.out.writeln("")
+            self.user_io.out.info("JSON file created at '%s'" % json_output)
+
+    def json_nodes_to_build(self, nodes_to_build, json_output, cwd):
+        data = [str(n) for n in nodes_to_build]
+        self._handle_json_output(data, json_output, cwd)
+
+    def _grab_info_data(self, deps_graph, grab_paths):
```

Comment:
I think this "conversion function" doesn't belong to this module. A `to_json` in the graph? Not very fan either...

Reply:
This class is created and called from outside the conan_api, the graph shouldn't have left the api. I'm sure that _this_ json should be the output of the `conan_api::info` function, but this would be a refactor (I vote for it) and not the objective of this PR.

Reply:
Agree. Open the engineering issue, please.